share/rollup.config.js

```diff
@@ -0,0 +1,21 @@
+import nodeResolve from '@rollup/plugin-node-resolve';
+
+export default {
+  input: 'dist/esm/index.js',
+  output: {
+    file: 'dist/plugin.js',
+    format: 'iife',
+    name: 'sharePlugin',
```

Comment:
I think we're using this style in other plugins, but it's certainly up for discussion

```suggestion
    name: 'capacitorShare',
```